### PR TITLE
BLOG: Team delivery improvements

### PR DESCRIPTION
**Pull request by choldgraf**
This is a short post to summarize the major changes to team practices and systems over the past three months. It covers both some of our "systems" for planning and delivery (like our boards) as well as changes to our practices and culture (like growing a culture of feedback and autonomy).

It is meant to give us a snapshot of our intentions, and help others understand the ways that we're making improvements. Hopefully it's also useful to other organizations looking for inspiration in how to improve their own coordination and delivery.

cc @haroldcampbell and @aprilmj in case either of you have feedback on this. I welcome comments or direct edits on any of the text. I'll leave the PR until Wednesday in case you (or others) have feedback, and will post it then if there aren't any objections.

### CODE REVIEW

**Review by haroldcampbell — APPROVED** [review]
lgtm

File: content/blog/2024/delivery-improvements/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+As a distributed organization, one of our biggest challenges has been the additional friction that comes with communicating across many time zones.
+It's much harder to solve a problem or get help when responses come with long gaps of time.
+
+As a result, we've defined several practices that will help our team members grow their autonomy and independence, while still moving towards the same targets and goals.
+This is ongoing work that we aim to continue developing over the next few months.
+
+**Personal growth**. 
+First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
+This began with our engineering team, but will continue with other areas of the organization as well.
+We believe that defining personal growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.
```
Comment (aprilmj) [inline content/blog/2024/delivery-improvements/index.md]: Could we change "personal growth" to "professional growth" or "professional development"? I'd like to highlight that the growth is focused on people's professional skills and qualities.

File: content/blog/2024/delivery-improvements/index.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+**Coaching**.
+Some members of our team have also begun benefiting from having regular 1:1 coaching to help their interpersonal development. Our team's coaching support includes work to empower improving self-awareness, overcoming self-limiting challenges and gaining technical leadership mastery.
+Coupled with our growing practices around team feedback, we aim to build a culture where team members have both internal and external support to help them grow.
+
+
+{{% callout %}}
+This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](https://www.linkedin.com/in/apriljohnson/).
```
Comment (aprilmj) [inline content/blog/2024/delivery-improvements/index.md]: My website is http://www.apriljohnson.io

**Review by aprilmj — APPROVED** [review]

File: content/blog/2024/delivery-improvements/index.md (hunk 1 above)
Comment (choldgraf) [inline content/blog/2024/delivery-improvements/index.md]:
```suggestion
**Professional growth**. 
First, we've started an audit of 2i2c's organizational roles and the responsibilities and the pathways for growth with each of them.
This began with our engineering team, but will continue with other areas of the organization as well.
We believe that defining professional growth trajectories will help identify the gaps in our team's skills that make it more difficult to act efficiently and independently.
```

File: content/blog/2024/delivery-improvements/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+lastmod: "2024-04-15"
+authors: ["Chris Holdgraf", "Harold Campbell"]
+tags: [culture]
+categories: [updates]
+featured: false
+draft: false
+---
+
+---
+
```
Comment (choldgraf) [inline content/blog/2024/delivery-improvements/index.md]:
```suggestion
```

File: content/blog/2024/delivery-improvements/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+**Coaching**.
+Some members of our team have also begun benefiting from having regular 1:1 coaching to help their interpersonal development. Our team's coaching support includes work to empower improving self-awareness, overcoming self-limiting challenges and gaining technical leadership mastery.
+Coupled with our growing practices around team feedback, we aim to build a culture where team members have both internal and external support to help them grow.
+
+
+{{% callout %}}
+This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](https://www.linkedin.com/in/apriljohnson/).
```
Comment (choldgraf) [inline content/blog/2024/delivery-improvements/index.md]:
```suggestion
This work was done in partnership with [April Johnson, a Transformation Consultant and Coach](http://www.apriljohnson.io/).
```

**Comment by choldgraf** [comment]
Thanks for the changes all, blog post is going up now!

**Pull request merged**